experiment/reporter.py

```diff
@@ -26,8 +26,9 @@
 from analysis import generate_report
 from analysis import data_utils
 
-CORE_FUZZERS_YAML = os.path.join(os.path.dirname(__file__), '..', 'service',
-                                 'core-fuzzers.yaml')
+CORE_FUZZERS_YAML = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..', 'service',
```

Comment:
This should just use utils.ROOT_DIR.

Reply:
Fixed in https://github.com/google/fuzzbench/pull/779